MoviesApp/MovieDetail/View/Cells/MovieRatingTableViewCell.swift

```diff
@@ -90,3 +79,16 @@ final class MovieRatingTableViewCell: UITableViewCell {
         ])
     }
 }
+
+// MARK: - MovieInfoCellProtocol
+
+extension MovieRatingTableViewCell: MovieInfoCellProtocol {
+    func configureCell(movieInfo: MovieInfo?) {
```

Comment:
```suggestion
    func configureCell(movieInfo: MovieInfo?) {
```
цел в экстеншен непонятно зачем и что это дает